boards/arduino-mega2560/board.c

```diff
@@ -56,7 +56,7 @@ void board_init(void)
 void SystemInit(void)
 {
     /* initialize UART_0 for use as stdout */
```

Comment:
The name of the folder is incorrect, we will try to change it since it obviously doesn't include all arduino boards, but only the smaller ones (328p kind and smaller).

I don't get why only the mega2560 has STDIO implementation, since all arduinos have afaik.